test/unit/parseGitmodulesDependencies.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import mock from 'mock-fs';
+import { parseGitmodulesDependencies } from '../../src/parseGitmodulesDependencies';
+
+describe('parseGitmodulesDependencies', () => {
+  before(() => {
+    mock.restore();
+  });
+  after(() => {
+    mock.restore();
+  });
+  describe('with no gitmodules file', function () {
+    before(() => {
+      mock({
+        '.gitmodules': '',
```

Comment:
Do we need the mock here? I suspect it will make it look like the file exists and is empty, and ideally we should test the case of the file not being present at all, in particular to ensure that the read call doesn't revert.

Reply:
good find :cowboy_hat_face: fixed it in cd631e6